extension/js/inspector/app/modules/Activity/views/ActivityGraph.js

```diff
@@ -1,19 +1,19 @@
 define(['backbone', 'marionette', 'app/modules/Activity/d3/graph', 'text!templates/devTools/activity/graph.html'
   ], function(Backbone, Marionette, Graph, tpl) {
 
-  var ActivityGraph = Backbone.Marionette.ItemView.extend({
+  var ActivityGraph = Backbone.Marionette.View.extend({
     
     template: tpl,
 
     tagName: "div",
 
     className: 'activity-graph',
 
-    defaults: {
```

Comment:
I'm not convinced that this was doing _anything_ at all ever.  defaults isn't an API on view.  plus what would setting this to `undefined` do anyways?  seems like this hash can be removed.